js/creator-23.js

```diff
@@ -3808,7 +3808,11 @@ async function addTextbox(textboxType) {
 }
 //ART TAB
 function uploadArt(imageSource, otherParams) {
-	art.src = imageSource;
+	if (!imageSource.includes('http')) {
```

Comment:
The real fix for this is to actually update `/creator/index.html` line 303 to use `onchange='imageURL(this.value, uploadArt, "autoFit");'` instead of the uploadArt function.